<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1499</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="Vert3">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/GreenFire.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="BateauAvale">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>170</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/Bateau2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="EauSas">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>250</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 164, 232);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Eau3">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>250</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 164, 232);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Rouge2_2">
    <property name="geometry">
     <rect>
      <x>1232</x>
      <y>328</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/RedFire.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Etap2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vert1_2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>330</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/GreenFire.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Terre">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>230</y>
      <width>671</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/Ecluse.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="BateauAmont">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>90</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/Bateau1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Vert2_2">
    <property name="geometry">
     <rect>
      <x>1232</x>
      <y>328</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/GreenFire.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Rouge4">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/RedFire.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Etap1">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>350</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vert4">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/GreenFire.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Etape1">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>350</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="Rouge3">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/RedFire.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Porte2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>140</y>
      <width>21</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Eau1">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>170</y>
      <width>275</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 164, 232);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Rouge1_2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>330</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/RedFire.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Porte1">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>140</y>
      <width>21</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>771</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <zorder>verticalLayoutWidget</zorder>
     <zorder>StopButton</zorder>
     <zorder>PorteDroite</zorder>
     <zorder>Bateau1</zorder>
     <zorder>Rouge2</zorder>
     <zorder>BordBas_2</zorder>
     <zorder>Authentification</zorder>
     <zorder>Vert1</zorder>
     <zorder>Rouge1</zorder>
     <zorder>BordBas</zorder>
     <zorder>PorteGauche</zorder>
     <zorder>label</zorder>
     <zorder>BateauMilieu</zorder>
     <zorder>Cabane</zorder>
     <zorder>Bateau2</zorder>
     <zorder>Vert2</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>30</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="Mdp">
       <item>
        <widget class="QTextEdit" name="Mdp2"/>
       </item>
       <item>
        <widget class="QPushButton" name="Mdp1">
         <property name="text">
          <string>Connection</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Authentification">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>0</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Authentification</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StopButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>270</y>
        <width>181</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : red ; border-raduis : 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="BordBas">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/BordBas.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="BateauMilieu">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">style =&quot;visible:hidden&quot;</string>
      </property>
      <property name="text">
       <string>BoatMilieu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Bateau2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>160</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../manuelle/images.qrc">
        <normaloff>:/pics/RessourcesGraph/Bateau2.png</normaloff>:/pics/RessourcesGraph/Bateau2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="PorteDroite">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>47</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/Porte.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="PorteGauche">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>47</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/Porte.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="Bateau1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../manuelle/images.qrc">
        <normaloff>:/pics/RessourcesGraph/Bateau1.png</normaloff>:/pics/RessourcesGraph/Bateau1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="Vert1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/GreenFire.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Rouge1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/RedFire.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="Cabane">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Rouge2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/RedFire.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Vert2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/GreenFire.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../RessourcesGraph/Place.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="BordBas_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../manuelle/images.qrc">:/pics/RessourcesGraph/BordHaut.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>Authentification</zorder>
     <zorder>StopButton</zorder>
     <zorder>BordBas</zorder>
     <zorder>BateauMilieu</zorder>
     <zorder>Bateau2</zorder>
     <zorder>PorteDroite</zorder>
     <zorder>PorteGauche</zorder>
     <zorder>Bateau1</zorder>
     <zorder>Vert1</zorder>
     <zorder>Rouge1</zorder>
     <zorder>Cabane</zorder>
     <zorder>Rouge2</zorder>
     <zorder>Vert2</zorder>
     <zorder>label</zorder>
     <zorder>BordBas_2</zorder>
     <zorder>label</zorder>
     <zorder>BordBas_2</zorder>
    </widget>
   </widget>
   <zorder>Vert3</zorder>
   <zorder>BateauAvale</zorder>
   <zorder>EauSas</zorder>
   <zorder>Eau3</zorder>
   <zorder>Rouge2_2</zorder>
   <zorder>Etap2</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>Vert1_2</zorder>
   <zorder>Terre</zorder>
   <zorder>BateauAmont</zorder>
   <zorder>Vert2_2</zorder>
   <zorder>Rouge4</zorder>
   <zorder>Etap1</zorder>
   <zorder>pushButton</zorder>
   <zorder>Vert4</zorder>
   <zorder>Etape1</zorder>
   <zorder>Rouge3</zorder>
   <zorder>Porte2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>Eau1</zorder>
   <zorder>Rouge1_2</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>Porte1</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1499</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../manuelle/images.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_Bateau_clicked()</slot>
 </slots>
</ui>
